.circleci/config.yml

```diff
@@ -155,6 +155,31 @@ jobs:
             just simulate-council
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
+
+  just_simulate_sep_007-presigned-pause:
```

Comment:
FP uses 007 in [this PR](https://github.com/ethereum-optimism/superchain-ops/pull/179), so this should probably be 008